queries/lua/injections.scm

```diff
@@ -16,7 +16,14 @@
   arguments: (arguments (string content: _ @query) .))
   (#any-of? @_vimcmd_identifier "vim.treesitter.query.set" "vim.treesitter.query.parse_query" "vim.treesitter.query.parse"))
 
-;; highlight string as query if starts with `;; query`
+; vim.rcprequest(123, "nvim_exec_lua", "return vim.api.nvim_buf_get_lines(0, 0, -1, false)", false)
+((function_call
+  name: (_) @_vimcmd_identifier
+  arguments: (arguments . (_) . (string) @_method . (string content: _ @lua)))
+  (#any-of? @_vimcmd_identifier "vim.rpcrequest" "vim.rpcnotify")
+  (#eq? @_method "\"nvim_exec_lua\""))
```

Comment:
hmm, is there any other methods that also needs lua highlighting?

Reply:
There's a number of new functions in 0.9.0; see the upstream queries.

Reply:
> There's a number of new functions in 0.9.0; see the upstream queries.

From the upstream lists, nvim-treesitter is missing `vim.api.nvim_cmd` and `vim.api.nvim_exec2`. @smjonas can you also add those into this PR?